app/assets/stylesheets/assignments.scss

```diff
@@ -158,6 +158,10 @@ input#assignment_repository_folder[readonly] {
       margin-bottom: 2em;
     }
   }
+
+  .group_options .group_button, .button_to{
```

Comment:
Selector should have depth of applicability no greater than 2, but was 3<br>Selector `group_options` should be written in lowercase with hyphens<br>Selector `group_button` should be written in lowercase with hyphens<br>Selector `button_to` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own single line<br>Opening curly brace `{` should be preceded by one space